src/Schema/Domain/Content/Worker/FieldDefinition/AddFieldDefinitionToDomainContentMutation.php

```diff
@@ -95,12 +101,21 @@ protected function getFieldDefinitionField(FieldDefinition $fieldDefinition): st
         return $this->getNameHelper()->fieldDefinitionField($fieldDefinition);
     }
 
-    private function getFieldDefinitionToGraphQLType(FieldDefinition $fieldDefinition, $operation): string
+    private function fieldType(array $args, $operation): string
```

Comment:
naming: this should still start with `get`  or something similar (I guess it's not a field type identifier, but identifier + string describing something, so the name should also reflect that.

Reply:
Yeah, but... it's a standard I've been following in the Workers. There is a lot of name building, that repeats a lot, and you end up with a looot of `get` calls.  See https://github.com/ezsystems/ezplatform-graphql/blob/master/src/Schema/Domain/Content/Worker/ContentType/AddContentOfTypeConnectionToDomainGroup.php#L57-L75 for an example.

I understand the remark, and to be honest, thought about the same, but I chose to favour consistency here.

Reply:
On second thought, I'll unify them with "get" in that package.

Reply:
Changed naming methods in that class. They now use the verb "name".